lib/fortnox/api/models/row/validator.rb

```diff
@@ -0,0 +1,19 @@
+require "fortnox/api/models/validator/base"
+
+module Fortnox
+  module API
+    module Validators
+      class Row < Fortnox::API::Validators::Base
+
+        attribute :account_number, inclusion: (0..9999)
+        attribute :article_number, size: 1..50
+        attribute :delivered_quantity, inclusion: (0..99_999_999_999_999)
+        attribute :description, size: 1..50
+        attribute :discount, inclusion: (0..999_999_999_999)
+        attribute :house_work_hours_to_report, inclusion: (0..99_999)
+        attribute :price, inclusion: (0..999_999_999_999)
```

Comment:
Organise these by type: presence, size, inclusion. Separate the different types by a blank line.

You should also be using `validates` and not `attribute` since the updated use of Hanami.